README.md

```diff
@@ -12,28 +12,44 @@ to ISO 4217 standards. The main items exported by the library are `Money` and `C
 
 ## Usage
 
-`Money` consists of an amount, which is represented by a Decimal type that it owns and a
-`Currency`, which is holds a reference to. `Currency` represents an ISO-4217 currency, and
- stores metadata like its numeric code, full name and symbol.
+`Money` consists of an amount and a currency. The amount is a Decimal type. The currency can be anything that 
+implements the `CurrencyType` trait. `IsoCurrency` is provided as an implementation of this trait which represents
+ISO-417 Currencies, along with all their metadata and formatting styles. `Currency` is also provided implementing this 
+trait which can be used to represent non-ISO currencies.
 
 ```rust
-// Money can be initialized in a few ways:
 use rusty_money::{money, Money, Currency};
+
+
+let v_bucks = Currency::new("VBX", 2)          // Create a US Dollar with an exponent of 2.
+Money::new(1, v_bucks)                         // One V Buck.
+Money::from_major(1, v_bucks)                  // One V Buck.
+Money::from_minor(100, v_bucks)                // One V Buck
+Money::from_str("2,000.00", v_bucks).unwrap(); // One V Buck
+
+// You call also call the money macro for convenience.
+money!(1, v_bucks)                          // One V Buck
```

Comment:
Why is this a macro and not just a function?

Reply:
Honestly, for convenience, but I think I'm going to deprecate it.